library/Backends/llvm/src/Juvix/Backends/LLVM/Codegen/Block.hs

```diff
@@ -163,22 +163,25 @@ module Juvix.Backends.LLVM.Codegen.Block
   )
 where
 
-import Data.ByteString.Short hiding (length)
-import Juvix.Backends.LLVM.Codegen.Types as Types
-import Juvix.Library hiding (Type, local)
-import qualified Juvix.Library.HashMap as Map
-import LLVM.AST
-import qualified LLVM.AST as AST
-import qualified LLVM.AST.CallingConvention as CC
-import qualified LLVM.AST.Constant as C
-import qualified LLVM.AST.Global as Global
-import qualified LLVM.AST.IntegerPredicate as IntPred
-import qualified LLVM.AST.Linkage as L
-import qualified LLVM.AST.Name as Name
-import qualified LLVM.AST.Operand as Operand
-import qualified LLVM.AST.ParameterAttribute as ParameterAttribute
-import qualified LLVM.AST.Type as Type
-import Prelude (String)
+import           Data.ByteString.Short                     hiding (length)
+import qualified Data.ByteString                           as BS
+import           Juvix.Backends.LLVM.Codegen.Types         as Types
+import           Juvix.Backends.LLVM.Codegen.Types.CString (CString)
+import qualified Juvix.Backends.LLVM.Codegen.Types.CString as CString
+import           Juvix.Library                             hiding (Type, local)
+import qualified Juvix.Library.HashMap                     as Map
+import           LLVM.AST
+import qualified LLVM.AST                                  as AST
+import qualified LLVM.AST.CallingConvention                as CC
+import qualified LLVM.AST.Constant                         as C
+import qualified LLVM.AST.Global                           as Global
+import qualified LLVM.AST.IntegerPredicate                 as IntPred
+import qualified LLVM.AST.Linkage                          as L
+import qualified LLVM.AST.Name                             as Name
+import qualified LLVM.AST.Operand                          as Operand
+import qualified LLVM.AST.ParameterAttribute               as ParameterAttribute
+import qualified LLVM.AST.Type                             as Type
+import           Prelude                                   (String)
```

Comment:
Are you using `stylish-haskell`(https://github.com/haskell/stylish-haskell)? Are these changes intended?

Reply:
I am, and yes. I thought that since I was already modifying the import section it would be fine to include the reformating in the PR, as it looks much more readable. Should I not do that?

Reply:
I fear there might be some conflict with Ormolu, which the CI runs when tests pass. If you have a strong opinion on this, feel free to bring it up in the next meeting

Reply:
I don't have a strong opinion, so I just reverted the changes.